README.md

```diff
@@ -1,66 +1,65 @@
+Check it out in action: https://www.youtube.com/watch?v=qZRcMQ6qIpg&t=6s&ab
 
 ## Development
 
-1. Build with Maven - mvn clean install
-1. Create a testing repository - for start you can fork my 
-https://github.com/xstefank/test-repo where the PR template is set up or 
-you can create your own :)
-    - For creating new repository check the github manuals/help
-    - PRs needs to be created to YOUR repository, you need to specify it 
-    under the base fork field in the Open a Pull Request page
-1. The project requires two properties to be set up to run correctly
-    - GitHub OAuth token - can be passed as a 
-      [MicroProfile Config](https://github.com/eclipse/microprofile-config)
-      property `tyr.github.oauth.token`
-    - Configuration file specifying the format of the PR - the example can 
-    be found in `tyr-runner/src/main/resources/format-example.yaml`, the supported
-      MicroProfile Config properties are `tyr.template.format.file` or
-      `tyr.template.format.url`
-1. start Tyr - `java -jar tyr-runner.jar` with above MP config properties set 
-   (as for instance system properties, e.g. `-Dtyr.template.format.file=/path/to/file`)
+1. Build Tyr with Maven in `tyr/tyr-runner/` folder using command `mvn clean install`
+1. Create a testing repository - for start you can fork 
+https://github.com/xstefank/test-repo where the Pull request template is already set up, or 
+you can create your own.
+    > - For creating new repository check the github manuals/help.
+    > - Pull requests needs to be made to **your** repository, therefore when you'll be doing Pull Request you need to specify this 
+    under the base fork field in the "Open a Pull Request" page.
+1. The project requires two [MicroProfile Config](https://github.com/eclipse/microprofile-config/blob/master/spec/src/main/asciidoc/microprofile-config-spec.asciidoc) properties to be set up to run correctly.  
+    1. `tyr.github.oauth.token` - Represents your unique GitHub OAuth token provided to Tyr.  
+        1. Go to your profile -> settings (top right) -> Developer Settings -> Personal access tokens  
+        1. Generate new token
+        1. Tick repo:status
+        1. Generate token
+        1. Use it as a value of this property, so the Tyr will be able to communicate with GitHub.
+   2. Next one is `tyr.template.format.file`, alternatively `tyr.template.format.url` - Represents path to configuration file specifying desired format of the Pull request. Example can be found in `tyr-runner/src/main/resources/format-example.yaml`
+    > Both properties can be passed to Tyr as:\
+        a. application properties - set once in `tyr-runner/src/main/resources/application.properties`. Necessary to rebuild Tyr after every adjustment of the file.\
+        b. system properties - passed as a parameter when starting Tyr. e.g. `-Dtyr.template.format.file=/path/to/file` (preferred option)\
+        c. CDI injection right into the code. 
```

Comment:
?? by default, there is also an option to pass MP config as Environment property but I have no idea what you mean by this.

Reply:
Could you provide any suggestions? This was meant just as an information for the user how the MC properties can be edited in case of insufficient experience with microprofile.

Reply:
c. as environment property TYR_TEMPLATE_FORMAT_FILE